### DISCUSSION TITLE
mix up of e.g. WriteCell and WriteHTML

### DISCUSSION THREAD

**Discussion by sscherger** [mpdf/mpdf#2022] [General]
I used TCPDF befor and will now use MPDF with HTML. But, would it be possible to use also the old code (for example for header and footer) beside the HTML? I tried this, but the HTML output was on a seperate page then.

**Comment by finwe** [comment]
No. mPDF is not TCPDF <s>even when it builds on top of it</s>. Deal with it.

**Reply by sscherger** [reply]
Thanks. I will change to HTML.
